### DISCUSSION TITLE
is there support punctuation characters.

### DISCUSSION THREAD

**Discussion by kamisch** [monkeytypegame/monkeytype#778] [Ideas] [answered]
Hello, 

I just started using MonkeyType recently and I love the site from the style to the ease of use. But it seems like there aren't any punctuation characters like periods, commas, or colons in the words. 

Is there a setting where I could turn on punctuations or its a design choice to not include it? 

Thanks

**Comment by ryan-harrington** [comment, marked as answer]
Yes. On the testing page, there is a setting in the top right corner to enable punctuation:

![image](https://user-images.githubusercontent.com/46948579/103403891-05460680-4b0f-11eb-98c2-c0f7ec93f39a.png)

**Reply by kamisch** [reply]
That's Awesome! Didn't know why I never realize that. 

Thanks

**Reply by ryan-harrington** [reply]
No problem.
